apps/src/applab/applab.js

```diff
@@ -1682,6 +1620,16 @@ Applab.setImageURL = function (opts) {
   return false;
 };
 
+Applab.playSound = function (opts) {
+  if (studioApp.cdoSounds) {
+    studioApp.cdoSounds.playURL(opts.url,
+                               {volume: 1.0,
+                                forceHTML5: true,
```

Comment:
Awesome utility, just a structure nit about the parameter passing—it feels like callers to the `playURL` method shouldn't have to know about the details of playback technologies & the fact that we normally don't attempt audio on HTML5.

Maybe we could collapse those two options `forceHTML5` and `allowHTML5Mobile` options into a single e.g. `isCrossDomain` option, or omit that and detect it by comparing the given URL and current browser address? (or, even simpler for the time being, assume all `playURL` calls are e.g. `playRemoteURL` calls)

Ideally the Sound(s) utility's public contract can be to act as a black box that does a "best attempt" at playback without halting execution